pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>fr.cnes.sonar</groupId>
     <artifactId>cnesreport</artifactId>
-    <version>3.1.0</version>
+    <version>3.1.8</version>
```

Comment:
Could you please revert this change in `pom.xml` too ?
We'd like to keep control on the version of the plugin for now.